packages/model/src/research-output.ts

```diff
@@ -261,7 +262,7 @@ export type ResearchOutputTeamResponse = ResearchOutputBaseResponse & {
 };
 
 export type ResearchOutputWorkingGroupResponse = ResearchOutputBaseResponse & {
-  workingGroups: [Pick<WorkingGroupResponse, 'id' | 'title'>];
```

Comment:
what's the reason behind this change? I believe we wanted it to be a 'tuple' so essentially an array of one, cause it should only ever be one WG, no?